config/archunit-store/stored.rules

```diff
@@ -0,0 +1,3 @@
+#
+#Sat Sep 10 22:27:26 IST 2022
+slices\ matching\ 'com.puppycrawl.tools.checkstyle.(**)'\ should\ be\ free\ of\ cycles=7999d2d1-606c-4e14-a852-2cb0fa661f48
```

Comment:
> 7999d2d1

Question: Are we required to use such naming?

Reply:
It was automatically generated by ArchUnit, didn't try changing it. Changed now, and it works perfectly fine.